### PR TITLE
Add APPLE techMob card

### PR DESCRIPTION
**Pull request by davidcanas**

### CODE REVIEW

**Comment by Sefiraat** [comment]
Given most other cards are for the production of raw/base materials, is it appropriate to have one for a fully finished, crafted, item like this?

**Comment by RelativoBR** [comment]
@davidcanas  Rejected because ready-made components are not the idea, just base features. If you separate only the suggestion of the apple, it will be welcome.

**Review by RelativoBR — CHANGES_REQUESTED** [review]
to remove CARD_ENERGY_CONNECTOR

**Comment by davidcanas** [comment]
done

File: src/main/java/com/github/relativobr/supreme/setup/SetupSimpleCard.java [hunk 1]
```diff
@@ -94,6 +96,6 @@ public static void setup(Supreme plugin) {
     TechGenerator.preSetup(plugin, SimpleCard.CARD_TIN_DUST, SlimefunItems.TIN_DUST, SlimefunItems.TIN_DUST);
 
     TechGenerator.preSetup(plugin, SimpleCard.CARD_ZINC_DUST, SlimefunItems.ZINC_DUST, SlimefunItems.ZINC_DUST);
-
+
```
Comment (Sefiraat) [inline src/main/java/com/github/relativobr/supreme/setup/SetupSimpleCard.java:99]:
```suggestion
```

**Review by Sefiraat — CHANGES_REQUESTED** [review]

File: src/main/java/com/github/relativobr/supreme/resource/mobtech/SimpleCard.java [hunk 2]
```diff
@@ -186,6 +190,6 @@ public class SimpleCard {
 
   public static final SlimefunItemStack CARD_ZINC_DUST = new SupremeItemStack("SUPREME_CARD_ZINC_DUST",
       Material.MUSIC_DISC_MELLOHI, "&bCard Machine Zinc Dust", "", "&fChange to Production Zinc Dust", "",
-      LoreBuilder.machine(MachineTier.MEDIUM, MachineType.MACHINE), "", "&3Supreme Component");
+      LoreBuilder.machine(MachineTier.MEDIUM, MachineType.MACHINE), "", "&3Supreme Component");
```
Comment (Sefiraat) [inline src/main/java/com/github/relativobr/supreme/resource/mobtech/SimpleCard.java:193]:
```suggestion
      LoreBuilder.machine(MachineTier.MEDIUM, MachineType.MACHINE), "", "&3Supreme Component");
```

**Review by Sefiraat — APPROVED** [review]
👍

File: src/main/java/com/github/relativobr/supreme/resource/mobtech/SimpleCard.java [hunk 3]
```diff
@@ -62,6 +62,10 @@ public class SimpleCard {
       "&bCard Machine Snow", "", "&fChange to Production Snow", "",
       LoreBuilder.machine(MachineTier.BASIC, MachineType.MACHINE), "", "&3Supreme Component");
 
+  public static final SlimefunItemStack CARD_APPLE = new SupremeItemStack("SUPREME_CARD_APPLE", Material.MUSIC_DISC_STRAD,
+      "&bCard Machine Snow", "", "&fChange to Production Apple", "",
```
Comment (Sefiraat) [inline src/main/java/com/github/relativobr/supreme/resource/mobtech/SimpleCard.java:66]: Whoops, saw this too late! 
```suggestion
      "&bCard Machine Apple", "", "&fChange to Production Apple", "",
```

**Review by Sefiraat — CHANGES_REQUESTED** [review]

File: src/main/java/com/github/relativobr/supreme/resource/mobtech/SimpleCard.java (hunk 3 above)
Comment (davidcanas) [reply src/main/java/com/github/relativobr/supreme/resource/mobtech/SimpleCard.java:66]: wtf

**Review by Sefiraat — APPROVED** [review]
👍

**Review by RelativoBR — APPROVED** [review]

**Pull request merged**